towncrier.pyproject.toml

```diff
@@ -1,6 +1,8 @@
 [tool.towncrier]
-    package = "allmydata"
-    package_dir = "src"
+# towncrier requires python3.5+ but Tahoe-LAFS doesn't support that
+# so we can't import it -- must specify --version to towncrier
+#    package = "allmydata"
+#    package_dir = "src"
```

Comment:
I'd leave this here with a reference to the towncrier ticket that added Python 2.7 support (certain to be released before Tahoe-LAFS supports Python >=3.5).  Or I'd delete it if we don't expect to actually return to use of the towncrier auto-find-version feature.  I think I'd prefer to return to use of that feature once we can.